package-lock.json

```diff
@@ -1,8 +1,8249 @@
 {
 	"name": "vscode-camelk",
 	"version": "0.0.32",
-	"lockfileVersion": 1,
+	"lockfileVersion": 2,
```

Comment:
the package-lock.json wasn't upgraded yet to a recent npm versiobn. it would have been better to do the upgrade in another PR as it will allow to review the changes. Fine for this time.